File: Week 2 - Basic Patterns/Lab 2/Tutor Demo - Average Days/AverageDays.py
month = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]

# sum pattern
sum = 0
for i in month:
    sum += i

average = sum / len(month)

# output pattern
print("Average days per month = " + str(average))

File: Week 2 - Basic Patterns/Lab 2/Tutor Demo - Average Days/AverageDays.java
public class AverageDays {

	//PUT YOUR CODE HERE;
	public static void main(String[] args) {
			double month[] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};

			// sum pattern
			double sum = 0;
			for (int i = 0; i < month.length; i++)
			{
				sum += month[i];
			}
			double average = sum / month.length;

			// output pattern
			System.out.println("Average days per month = " + average);
	}
}
